### PR TITLE
pyHyp options were explained for airfoil mesh generation

### PR DESCRIPTION
**Pull request by ArshSaja**
## Purpose
See title

## Type of change
- Documentation update

## Testing
Explain the steps needed to test the new code to verify that it does indeed address the issue and produce the expected behavior.

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: machAeroTutorials/airfoilopt_mesh.rst
Comment (marcomangano) [inline machAeroTutorials/airfoilopt_mesh.rst:58]: I would swap the sentence order, because we just set the symmetry here. Then you add that by default the `k=1` (or `kLow`?) is set by default to be a non-slip wall.

**Review by marcomangano — CHANGES_REQUESTED** [review]
Thanks for fixing this! Please address the comment, then I will merge the PR.

**Comment by marcomangano** [comment]
@ArshSaja why did you close this and opened a new one? It's the same thing right?

**Comment by ArshSaja** [comment]
Yes it is the same thing. I wasn't able to do PR again. Sorry if it is wrong

**Comment by nwu63** [comment]
When you push to a branch which is part of a PR, the PR will automatically update so there is no need to close and open another one. I am going to re-open this one and close the other.

**Comment by ArshSaja** [comment]
Ok sorry for this. I didn't know that

**Comment by marcomangano** [comment]
Looks good to me now, thanks again! Maybe @anilyil can take a quick look before we merge it

File: machAeroTutorials/airfoilopt_mesh.rst
Comment (sseraj) [inline machAeroTutorials/airfoilopt_mesh.rst:58]: I think it should be just ``kLow`` not ``kLow=1``?

Comment (marcomangano) [reply machAeroTutorials/airfoilopt_mesh.rst:58]: thanks for catching it!

Comment (sseraj) [reply machAeroTutorials/airfoilopt_mesh.rst]: I was looking into the ``BC`` option as part of [this issue](https://github.com/mdolab/pyhyp/issues/27). I found that there is actually no ``kLow`` or ``kHigh`` option. Also, I wouldn't say it's no-slip by default because you can use the same mesh for Euler runs. I think it's more accurate to say that the input surface mesh (which corresponds to ``kLow``) is automatically assigned to be a wall.

Comment (marcomangano) [reply machAeroTutorials/airfoilopt_mesh.rst]: Very good point... We should stay as general as possible! Would you mind edit this really quick? Otherwise I might do this tomorrow

Comment (sseraj) [reply machAeroTutorials/airfoilopt_mesh.rst]: I just realized that I can't push to this branch.

As a broader issue, we should discuss what to document in these tutorials vs. what to document in the code repos. There are some options documented here that are not documented in `pyHyp`. There is also duplication in this repo for the airfoil and wing cases. After this PR, the airfoil mesh generation page will have details that the wing mesh generation page does not.

Comment (ArshSaja) [reply machAeroTutorials/airfoilopt_mesh.rst]: Do you want me to edit and push it ?

Comment (sseraj) [reply machAeroTutorials/airfoilopt_mesh.rst]: That would be great.

Comment (ArshSaja) [reply machAeroTutorials/airfoilopt_mesh.rst:58]: Okay I did that!

Comment (sseraj) [reply machAeroTutorials/airfoilopt_mesh.rst:58]: Just ``kLow`` was correct. Marco already made that edit. I would replace the entire second sentence with "The input surface is automatically assigned to be a wall boundary."

Comment (marcomangano) [reply machAeroTutorials/airfoilopt_mesh.rst:58]: I agree on the "misallocation" of the info throughout the codes and the repos @sseraj . I was looking for some of these options on `pyHyp` too and got a bit confused. We probably need a major check of the tutorial and the related repos,.. yet another problem connected to the bigger "knowledge transfer" issue. We need to discuss this soon with the maintainers for every repo!

**Review by marcomangano — APPROVED** [review]

File: machAeroTutorials/airfoilopt_mesh.rst
Comment (nwu63) [reply machAeroTutorials/airfoilopt_mesh.rst]: There is bound to be some duplicated information, but the most comprehensive and general ones (such as documenting the options) should be in the repository docs, and I think in most cases it's sufficient to point readers to those pages without duplicating the explanation. The information here should be specific to this particular case.

Comment (sseraj) [reply machAeroTutorials/airfoilopt_mesh.rst]: I agree. I will add the ones here to the pyHyp options table.

**Comment by nwu63** [comment]
Are we waiting for anything before we merge this? If not then one of the reviewers can merge it.

**Review by sseraj — APPROVED** [review]
I'm not a maintainer but this looks fine. Please merge @nwu63

**Comment by nwu63** [comment]
One last thing, unrelated to the PR but @ArshSaja you committed on the master branch again, so you will probably have to reset your branch to make it consistent with `mdolab/master`. In the future please make a new branch before committing.

**Pull request merged**
